filament/src/details/View.h

```diff
@@ -531,6 +534,7 @@ class FView : public View {
     Viewport mViewport;
     bool mCulling = true;
     bool mFrontFaceWindingInverted = false;
+    bool mIsTransparentPickable = false;
```

Comment:
nit: `mIsTransparentPickingEnabled` for consistency.

Reply:
Done.